packages/uniorg-parse/src/parser.ts

```diff
@@ -1466,7 +1532,8 @@ class Parser {
     const contentsBegin = this.r.offset() + m.index + m[1].length + m[3].length;
     const contentsEnd = contentsBegin + m[4].length;
     this.r.resetOffset(contentsEnd + 1);
-    return u('code', { value }, []);
+    const [begin, end] = this.getCurrentRange(value);
```

Comment:
❗🐛 `getCurrentRange` assumes that cursor is the beginning of the element. It's at the end after `this.r.resetOffset(contentsEnd + 1)`